source/runbooks/dr-process.html.md.erb

```diff
@@ -0,0 +1,362 @@
+
+- If the issue is across all accounts, login to the Modernisation Platform account, and raise the highest level severity ticket, whilst also engaging with our AWS representative in the `#ext-aws` channel.
+
+
+- If this issue is in one account, login to that account, and raise the highest level severity ticket if production, and also engaging with our AWS representative in the `#ext-aws` channel.
+
+
+- If this occurs during working hours, create a huddle / meet. Outside of hours, if you have been called out, call the second member of staff that is on call and post to the channel. With the PagerDuty alerts, incident channels will be created. If there are too many channels to follow, make a lead channel to coordinate.
+
+
+## Internal Escalation
+
+
+**Simon can you help fill in this**
+
+
+Inform the following channels of the situation and put forward a contactable member of staff from our team.
+
+
+`#ask-modernisation-platform`
+
+
+`#modernisation-platform`
+
+
+`#modernisation-platform-alerts`
+
+Following [this]( https://user-guide.modernisation-platform.service.justice.gov.uk/runbooks/manage-an-incident.html#incident-process) process.
+
+## Disaster Recovery Types
+
+
+**Let’s work the problem, people. Let’s not make things worse by guessing.**
+
+
+- London has been lost but **no services need to migrate**. [Click here](#london-has-been-lost-but-returning-soon).
+- London has been lost, and **all services need to migrate**. [Click here](#disaster-recovery-running-outside-of-london).
+- Loss of a **single account**. [Click here](#single-account).
+
+
+## London has been lost but returning soon.
+
+
+### Getting a decision
+
+
+Once a ticket has been raised with AWS, and we have engaged with them and have total confirmation that the infrastructure will be returning within X number of minutes or hours, we will need to pass this information on for an informed decision to be made above.
+
+
+Only when we have this, can we update users on what is next.
+
+
+### Priority list
+
+
+If there is limited availability, which is a possibility, these are the priority accounts that need to be checked before we can focus on member-accounts.
+
+
+
+|Account|Information|
+|----------------- |------------------------------------------------ |
+|Core-Shared-Services|Creates the core shared services account resources. |
+| Core-Network-Services | Creates the core networking account resources; this is the highest priority. |
+|Core-Logging|Creates the core logging account resources |
+|Core-Security|Creates the core security account resources |
+|Cloud-Platform-Transit-Gateways|Across all regions. |
+|Core-VPC-*|Creates the core VPC resources in the VPC accounts, needed after the core-networking-services. |
+| Modernisation-Platform | Creates key resources such as S3 state buckets for the platform |
+
+
+### Test Accounts
+
+
+If there is limited availability, power off all infrastructure in the test accounts. All are useless until the platform is running.
+
```

Comment:
I think it would be good to add a note, that if the decision it to rebuild then follow the [Running outside of London DR steps](#disaster-recovery-running-outside-of-london).

Reply:
Where?